source/manual/troubleshooting-vagrant.html.md

```diff
@@ -146,14 +146,33 @@ directory on your host, and then run `govuk_puppet` again in the VM.
 The other solution, as mentioned in
 [this GitHub issue against Vagrant](https://github.com/mitchellh/vagrant/issues/8061)
 is to force macOS to refresh the NFS cache. To do this on your host you run
-`ls -alR > /dev/null` in the root of your govuk folder. To do this on your VM
-you run `find /var/govuk -type d -exec touch '{}'/.touch ';' -exec rm -f '{}'/.touch ';' 2>/dev/null`.
+
+```bash
+ls -alR > /dev/null
+```
+
+in the root of your govuk folder. To do this on your VM
+you run
+
+```bash
+find /var/govuk -type d -exec touch '{}'/.touch ';' -exec rm -f '{}'/.touch ';' 2>/dev/null
+```
```

Comment:
being a touch anal, but since most blocks use `shell` we should probably be consistent through this doc. I don't know if using shell or bash do things much differently though highlighting wise.